### ISSUE TITLE
[Dataset] <Stirrup Rebar Instance Segmentation for Grasping>

### ISSUE THREAD

**Issue by tsrobcvai** [ruoxinx/OpenConstruction-Datasets#6]
### Dataset name

Stirrup Rebar Instance Segmentation for Grasping

### Release year

2025

### Dataset URL / Access link

https://huggingface.co/datasets/tsrobcvai/Synthetic_Dataset_for_Stirrup_Rebar_Segmentation

### Paper (if available)

Rebar grasp detection using a synthetic model generator and domain randomization

### Authors

Tao Sun, Beining Han, Szymon Rusinkiewicz, Yi Shao

### DOI / URL

https://doi.org/10.1016/j.autcon.2025.106252

### Short description

The dataset contains:

A synthetic training set of 12,000 images, and a synthetic validation set of 4,000.
A synthetic test set of 4,800 images (only top rebars are annotated).
A real-world test set of 233 images (only top rebars are annotated).
Diverse rebar specifications, stacking, lighting, distractors, and background conditions.

### Data modality

RGB images

### Object classes (comma separated)

1

### Potential tasks (comma separated)

Stirrup rebar grasping
